### PR TITLE
RA-1816: Return of boolean expressions should not be wrapped into an …

### PR DESCRIPTION
**Pull request by kdaud**
see https://issues.openmrs.org/browse/RA-1816

### CODE REVIEW

File: omod/src/main/java/org/openmrs/module/referenceapplication/page/controller/LoginPageController.java [hunk 1]
```diff
@@ -355,10 +355,7 @@ public String post(@RequestParam(value = "username", required = false) String us
 	 * @return
 	 */
 	private boolean isSelectLocationRequest(boolean isUserLocPropEnabled) {
-		if (isUserLocPropEnabled && Context.isAuthenticated() && Context.getUserContext().getLocationId() == null) {
-			return true;
-		}
-		return false;
+		return (isUserLocPropEnabled && Context.isAuthenticated() && Context.getUserContext().getLocationId() == null);
```
Comment (HerbertYiga) [inline omod/src/main/java/org/openmrs/module/referenceapplication/page/controller/LoginPageController.java:358]: cc @sacull

Comment (sacull) [inline omod/src/main/java/org/openmrs/module/referenceapplication/page/controller/LoginPageController.java:358]: Why not?
```suggestion
		return isUserLocPropEnabled && Context.isAuthenticated() && Context.getUserContext().getLocationId() == null;
```

Comment (kdaud) [reply omod/src/main/java/org/openmrs/module/referenceapplication/page/controller/LoginPageController.java]: Have addressed the comment and updated the PR

**Comment by sacull** [comment]
It looks good to me now. ;)

**Comment by dkayiwa** [comment]
@sacull just given you write access to this repository and the corresponding JIRA ticket. So feel free to do the needful. 😊

**Comment by sacull** [comment]
Not again. You don't like me, @dkayiwa, if you keep giving me more work. ;)

**Comment by dkayiwa** [comment]
@sacull you made me laugh my head off! Loool. 🤣 
I promise to treat you better next time. 😆

**Pull request merged**